tests/airgapped/README.md

```diff
@@ -0,0 +1,91 @@
+# Testing Airgapped Installation
+
+For running the tests we expect an environment that can:
+1. Spin up LXC containers
+2. Have Docker, to pull images
+
+We need docker to pull all the images necessary and push them to the airgapped
+lxc container.
+
+## Setup
+
+We've prepared some scripts for setting up the environment
+```bash
+./tests/airgapped/setup/setup.sh
+```
+
+We've observed that in almost all cases we needed to reboot to be able to
+run docker as sudo and the lxc container to, initially, have network access.
+
+## Running the tests
+
+You can run the script that will spin up an airgapped microk8s cluster with:
+
+```bash
+./tests/airgapped/airgap.sh \
+  --node-name airgapped-microk8s \
+  --microk8s-channel 1.24/stable
+  --bundle-path releases/latest/edge/bundle.yaml
+```
+
+### Running with a subset of images
+
+By default, if no `images.tar.gz` file is found, the script will try to download
+all the CKF images. These are 125Gb, which will make it difficult for running a
+lot of tests locally.
```

Comment:
I think skopeo:
1. can copy a remote image directly to a local registry
2. is smart enough to skip if already exists

Reply:
@sed-i what would be the benefits of using skopeo instead of docker?